src/handlers/mxapp.js

```diff
@@ -13,7 +13,21 @@ const dynamoName = nconf.get("DYNAMODB:TABLE:MXCONTROL");
 const msgMXControlRoom =
     async message => hipchat.notify(message, {room: "MXControl"});
 
-const handleMXAppControlResponse = async ({source, message}) => {
+const handleMXAppResponse = async({source, message}) => {
+
+    if (source == "MXControl")
+        return await handleMXControlCronResponse(message);
+
+    //else
+    return await handleMXControlStartupResponse(source);
+};
+
+const handleMXControlCronResponse = async(message) => {
```

Comment:
this `async/await` does nothing

Reply:
Ordering of MXControl messages.

However, to reference a previous comment of yours - ordering is moot if you send everything as a single well formatted large message.